PythonScript/src/MenuManager.cpp

```diff
@@ -936,7 +939,7 @@ tstring MenuManager::formatMenuName(const TCHAR *name)
 
 idx_t MenuManager::findMenuCommand(HMENU hParentMenu, const TCHAR *parentMenuName, const TCHAR *menuName, const TCHAR *menuOption)
 {
-	size_t iMenuItems = (size_t)GetMenuItemCount(hParentMenu);
+	int iMenuItems = GetMenuItemCount(hParentMenu);

```

Comment:
As GetMenuItemCount() could return -1 in case of an error the for loop below with unsigned idx_t could create a problem.